ci-operator/step-registry/cco/remove-root-secret/cco-remove-root-secret-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+function remove_root_secret()
+{
+    local secret_name=$1
+    echo "Removing secret ${secret_name} from kube-system"
+    oc delete secret -n kube-system ${secret_name}
```

Comment:
append `|| return 1` ?

Reply:
```
$ oc delete secret -n kube-system aaa
Error from server (NotFound): secrets "aaa" not found
$ echo $?
1
```

Reply:
I will add once rehearse job is done.

Reply:
The following command is a echo command, so that means if the deletion failed, would return 0 from the echo command, once that, the caller would treat that as a good behavior, sounds not right.

Reply:
> I will add once rehearse job is done.

Got it.